Sources/Crypto/RSA/RSAKey.swift

```diff
@@ -39,6 +39,29 @@ public struct RSAKey {
         self.type = type
         self.c = key
     }
+    
+    /// Creates a new `RSAKey` from components.
```

Comment:
A small example on how to use this API in the doc block would be great. 

See [this](https://github.com/vapor/crypto/blob/master/Sources/Crypto/Digest/Digest.swift#L106-L118) doc block for an example.

Reply:
@tanner0101 done!